src/Orleans/Runtime/OutsideRuntimeClient.cs

```diff
@@ -116,6 +116,7 @@ public OutsideRuntimeClient(ClientConfiguration cfg, bool secondary = false)
             services.AddFromExisting<IInternalGrainFactory, GrainFactory>();
             services.AddFromExisting<IGrainReferenceConverter, GrainFactory>();
             services.AddSingleton<MessageFactory>();
+            services.AddSingleton<Func<string, Logger>>(LogManager.GetLogger);
```

Comment:
Yeah, I think it's better this way. It's not very discoverable, but this signature makes sense to me unless we want to maybe add some explicit logging provider interface, like `ILogManager`. We eventually may, but this is a conservative option for now, IMO. What do you think?